content/go/concepts/switch/switch.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: 'Switch'
+Description: 'A switch statement allows the selective execution of one of several blocks of code based on the value of an expression.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Control Flow'
+  - 'Switch'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+A `switch` statement allows the selective execution of one of several blocks of code based on the value of an expression. It can be thought of as an expanded `if ... then ... else` statement, with many potential `then` clauses.
+
+## Syntax 1
+
+This is the typical `switch` statement syntax:
+
+```pseudo
+switch <expression> {
+  case <value 1>:
+    <statements>
+  case <value 2>:
+    <statements>
+  .
+  .
+  .
+  case <value N>:
+    <statements>
+  default:
+    <statements>
+}
+```
+
+When `<expression>` is equal to the value of one of the `case` statements, the corresponding `<statements>` block is executed. If the `<expression>` matches none of the `case` values, then the optional `default` `<statements>` block is executed.
```

Comment:
What happens if there is no `default` block? Will it cause an error or something?